### ISSUE TITLE
Use "import six.moves.cPickle as pickle",  error:"ImportError: cannot import name 'moves' from 'pickle'"

### ISSUE THREAD

**Issue by sdyzw** [Nuitka/Nuitka#2405]
>1.7
Commercial: None
Python: 3.8.3 (tags/v3.8.3:6f8c832, May 13 2020, 22:37:02) [MSC v.1924 64 bit (AMD64)]
Flavor: CPython Official
Executable: F:\Virtualenv\NuitkaPackV1\Scripts\python.exe
OS: Windows
Arch: x86_64
WindowsRelease: 10

Just pack that sentence
```
# main_a.py
import six.moves.cPickle as pickle
```
This is pack command
```
nuitka --mingw64 --show-progress --show-memory  main_a.py --standalone
```

This is error info:
```
Traceback (most recent call last):
  File "E:\xxxxx\PACKRE~1\main_a.py", line 2, in <module>
ImportError: cannot import name 'moves' from 'pickle' (E:\xxxxx\PACKRE~1\pickle.py)
```
I tried multiple versions with this error,

**Comment by kayhayen** [comment]
Something about the import resolving is going wrong and what is supposed to turn `six.moves.cPickle` to `pickle` turns it a mess, where moves is imported from there, and then from that it would import cPickle, which is just wrong in amazing ways.

**Comment by kayhayen** [comment]
Ok, deep down, this does the following, it imports `six.moves.cPickle` which is then actually as a value `six`, and then it looks up `moves` and `cPickle` to get the value needed to assign the `as` for it, since import `a.b` gives `a` and in the syntax however, the last level is used instead. So, we are doing this kind of late for the re-formulation of imports, and become incompatible by stripping the sub-package stuff.

**Comment by kayhayen** [comment]
Thanks for taking the time. I did not replace module names during parsing, because imports that come into existance later should also apply them, and if I do it only there, then I know that to be the case. However, this shows that while for those it is still good to do, actually it's best to do it as soon as possible.

**Comment by kayhayen** [comment]
Thanks for your report, this is fixed on the factory branch, which is a development version under rapid development. You can try it out by going here: https://nuitka.net/doc/factory.html 

Feedback if this is working is very welcome, just please do not share plans of doing it, but rather confirmations or denials of it working.

**Comment by sdyzw** [comment]
> Thanks for your report, this is fixed on the factory branch, which is a development version under rapid development. You can try it out by going here: https://nuitka.net/doc/factory.html
> 
> Feedback if this is working is very welcome, just please do not share plans of doing it, but rather confirmations or denials of it working.

Thanks for your reply, I have solved this problem with 1.8.

**Comment by kayhayen** [comment]
Part of the 1.8 release I just made.
